roles/crawler/tasks/configure-databases.yml

```diff
@@ -99,6 +110,12 @@
       lc_ctype: en_US.UTF-8
       template: "{{ fpsd_database_psql_env.PGDATABASE }}"
 
-  when: raw_schema_result|changed or raw_schema_tables_result|changed
+  - name: Setup TABLEFUNC extension.
+    postgresql_ext: name=tablefunc db="test{{ fpsd_database_psql_env.PGDATABASE }}"
+    register: postgres_test_extension
+    always_run: true
+    changed_when: false
+
+  when: raw_schema_result|changed or raw_schema_tables_result|changed or features_schema_result|changed
```

Comment:
To reflect our desire to know if the presence of the tablefunc extension has changed, as well as our combining the schemata creation into one task, we would change this to:
```
  when: raw_schema_tables_result|changed or postgres_extension|skipped or schemata_results|skipped
```